jsonapi/helpers.go

```diff
@@ -86,7 +86,16 @@ func Singularize(word string) string {
 func GetTagValueByName(tfield reflect.StructField, name string) string {
 	str := tfield.Tag.Get("jsonapi")
 	if str == "" {
-		return ""
+		str = tfield.Tag.Get("json")
+		if str == "" || str == "-" {
+			return ""
+		}
+
+		if idx := strings.Index(str, ","); idx != -1 {
```

Comment:
That is actually the spec for the json encode tags. They do not follow the same convention as api2go.